components/automate-ui/src/app/pages/service-groups/service-groups.component.html

```diff
@@ -191,7 +191,7 @@ <h3>Loading service groups...</h3>
   -->
   <div class="empty-status" *ngIf="(serviceGroupsStatus$ | async) === 'loadingSuccess' && sgHealthSummary.total === 0">
     <h3>Add service groups</h3>
-    <p>Contact your account representative to find out more about adding service groups.</p>
+    <p>Find out more about <a href="https://automate.chef.io/docs/applications-setup/">adding service groups.</a></p>
```

Comment:
FYI: The link is not expected to work until the next time we promote acceptance to current. Chef Software employees can view the acceptance version of the page over VPN at https://a2-docs-acceptance.cd.chef.co/docs/applications-setup/

Reply:
should it open in a new tab? or direct away from automate?

Reply:
Good point. I couldn't find any of Liz' original designs for this but I'm assuming it should be a new tab. Updating.